### PR TITLE
Getsstables

### PR DESCRIPTION
**Pull request by sylvestor88**
Command to list sstables in a node

### CODE REVIEW

**Comment by ptnapoleon** [comment]
Can you squash these commits?

File: ccmlib/node.py [hunk 1]
```diff
@@ -812,6 +812,13 @@ def run_sstablemetadata(self, output_file=None, datafiles=None, keyspace=None, c
         if output_file == None:
             return results
 
+    def run_sstables(self, output_file=None, datafiles=None, keyspace=None, column_families=None):
```
Comment (ptnapoleon) [inline ccmlib/node.py]: I'd prefer if the method name contained `getsstables` or `get_sstables`

File: ccmlib/cmds/node_cmds.py [hunk 2]
```diff
@@ -459,6 +460,40 @@ def run(self):
...
+    def description(self):
+        return "Run getsstables to get sstables of this node"
+
+    def get_parser(self):
+        usage = "usage: ccm node_name getsstables [options] [file]"
+        parser = self._get_default_parser(usage, self.description())
+        parser.add_option('-k', '--keyspace', type="string", dest="keyspace", default=None,
+                          help="The keyspace to use [use all keyspaces by default]")
+        parser.add_option('-c', '--column-families', type="string", dest='cfs', default=None,
+                          help="Comma seperated list of column families ut use (requires -k to be set)")
```
Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: It's spelled `separated`

Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: Also, check out the typo at `families ut use`

File: ccmlib/cmds/node_cmds.py [hunk 3]
```diff
@@ -459,6 +460,40 @@ def run(self):
                                    column_families=self.column_families, size=self.size,
                                    no_snapshot=self.no_snapshot)
 
+class NodeGetsstablesCmd(Cmd):
+    def description(self):
+        return "Run getsstables to get sstables of this node"
```
Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: Could you make this more descriptive? Are we returning paths? Just file names?

File: ccmlib/cmds/node_cmds.py (hunk 2 above)
Comment (sylvestor88) [inline ccmlib/cmds/node_cmds.py]: This one I just picked from NodeSstablesplitCmd(Cmd) so didn't look much at the grammar. Have corrected that too now.

**Pull request closed without merging**